### PR TITLE
Run PosePlot from entirely static assets

### PR DESCRIPTION
**Pull request by broadwell**
The data preparation (creating static assets = images, atlases, JSON metadata and layout files) for the PosePlot "Explorer" viz tab is now performed in the same step as the pose clustering process, ensuring consistency across visualizations.

All of the Python code necessary for the PosePlot/PixPlot data generation has been extracted into a single script, so it is no longer necessary to install and run the `pixplot` Python package. The front-end PixPlot code has been further extracted/vendored. As a consequence, almost certainly not necessary to have a separate `web-extras` container anymore, nor to run the PosePlot page in its own server, but on the other hand it's kind of nice to keep it at arm's length from the rest of the project.

One consequence of the change in the first paragraph above is that there's now too much overlap between the `cluster-poses` and the `cluster-plot-poses` workflows, but that can be DRYed up later.

The PR also includes some code refactoring, file/folder moves, and some rudimentary attempts to use the `image-cache` folder more effectively.

### CODE REVIEW

File: api/cluster_video_poses.py [hunk 1]
```diff
@@ -210,7 +207,7 @@ async def main() -> None:
     # Get the full pose data for each representative movelet from a track in a cluster,
     # to be used to display armatures
 
-    filtered_movelet_counts = dict()
+    filtered_movelet_counts = {}
```
Comment (simonwiles) [inline api/cluster_video_poses.py:213]: Out of interest, did a linter nudge you towards this?  It's stylistic only, I think?  (I personally tend to prefer the dict literal to the constructor, but I don't think I could tell you why...)

**Review by simonwiles — APPROVED** [review]
Cool, let's get this in 👍️

File: api/cluster_video_poses.py (hunk 1 above)
Comment (broadwell) [reply api/cluster_video_poses.py:213]: Yeah, looks like it's this rule, as cited by Ruff:
https://docs.astral.sh/ruff/rules/unnecessary-collection-call/
Seems likely to be a pretty small performance hit, and an even smaller chance that someone would have the chutzpah to bind `dict()` to something else, but there you have it.

File: api/cluster_and_plot_poses.py [hunk 2]
```diff
@@ -292,10 +292,9 @@ async def main() -> None:
                     bbox = [min_x, min_y, b_w, b_h]
 
             frame_image = Path(
-                "mime-cache", str(video_id), "frames", f"{target_frame}.jpeg"
+                "/static", str(video_id), "frames", f"{target_frame}.jpeg"
```
[the remark this answers is not in the document]
Comment (broadwell) [reply api/cluster_and_plot_poses.py:295]: OK yeah I saw this being done in `server.py` and considered adding it here, but I was working quickly and also discouraged by uncertainty about whether calling `load_dotenv()` would be needed here, so I didn't pursue it. It's probably something to add in the next iteration, though.

**Pull request merged**